wpilib/wpilib/builtinaccelerometer.py

```diff
@@ -88,3 +88,6 @@ def initSendable(self, builder):
         builder.addDoubleProperty("X", self.getX, None)
         builder.addDoubleProperty("Y", self.getY, None)
         builder.addDoubleProperty("Z", self.getZ, None)
+
+    def getSimObject(self):
```

Comment:
Hm?

Reply:
I don't know what it is

Reply:
We should probably add that commit (`337e89cf6e791d67f6e80fbba8c613cc98d2a36a`) to the git source-track ignore file... thoughts @virtuald?